cmd/virtual-workspaces/options/options.go

```diff
@@ -29,7 +29,7 @@ import (
 	genericapiserveroptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/component-base/logs"
 
-	workspacesoptions "github.com/kcp-dev/kcp/pkg/virtual/workspaces/options"
+	rootoptions "github.com/kcp-dev/kcp/pkg/virtual/framework/rootapiserver/options"
```

Comment:
I think this is the wrong direction of a change. Having options in virtual/workspaces was correct. We need virtual/syncer/options and wire those where we have Workspaces today.